js/script.js

```diff
@@ -51,23 +51,23 @@ $(function(){
 		ignoreScroll: 0,
 		$container: $('#container'),
 		lastDateGroup: null,
-		$content: $('#app-content'),
+		$content: $(document),
 		firstKnownId: 0,
 		lastGivenId: 0,
 		activities: {},
 
 		prefill: function () {
 			this.ignoreScroll += 1;
-			if (this.$content.scrollTop() + this.$content.height() > this.$container.height() - 100) {
+			if (this.$content.scrollTop() + $(window).height() > this.$container.height() + 100) {
```

Comment:
I will make it -100 again, so new activities are already loaded, before you reach the end, so ideally you never see the loading circle

Reply:
Ah, make sense now. I couldn't figure why it was set to -100!